roles/robocup2021.role

```diff
@@ -0,0 +1,45 @@
+nuclear_role(
+  # FileWatcher, FileLogHandler and Signal Catcher Must Go First. KinematicsConfiguration usually goes after these and
+  # without it many roles do not run
+  extension::FileWatcher
+  support::SignalCatcher
+  support::logging::FileLogHandler
+  # This must come first as it emits config which many roles depend on (e.g. SensorFilter, WalkEngine)
+  motion::KinematicsConfiguration
+  # Support
+  support::configuration::GlobalConfig
+  network::NUClearNet
+  # support::NUsight
+  support::configuration::SoccerConfig
+  # input
+  input::GameController
+  # Simulator connection
+  platform::Webots
+  input::SensorFilter
```

Comment:
For some reason webots needs to be after sensorfilter, or it wont work. Was probably the issue in the lab today.